src/index.html

```diff
@@ -11,7 +11,213 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      onsubmit="onSubmit()">
+
+      <fieldset class="block">
+        <legend class="chapter">
+          Personal information
+        </legend>
+        <div class="field_title">
+          <label for="surname">
+            Surname:
+            <input
+              type="text"
+              autocomplete="off"
+              class="field"
+              name="Surname"
+              id="surname"
+              required>
+          </label>
+        </div>
+        <div class="field_title">
+          <label for="name">
+            Name:
+            <input
+              type="text"
+              autocomplete="off"
+              class="field"
+              name="Name"
+              id="name"
+              required>
+          </label>
+        </div>
+        <div class="field_title">
+          <label for="age">
+            How old are You?
+            <input
+              name="age"
+              type="number"
+              class="field"
+              value="12"
+              min="10"
+              max="100"
+              id="age">
+          </label>
+        </div>
+        <div class="field_title">
+          <label for="dob">
+            Full date of birth:
+            <input
+              name="date of birth"
+              type="date"
+              class="field"
+              id="dob">
+          </label>
+        </div>
+        <div class="field_title">
+          <label for="terms">
+            I accept the term of the agreement
+            <input
+              name="terms"
+              type="checkbox"
+              class="field"
+              id="terms"
+              required>
+          </label>
+        </div>
+      </fieldset>
+      <fieldset class="block">
+        <legend class="chapter">
+          Registration
+        </legend>
+        <div class="field_title">
+          <label for="mail">
+            E-mail:
+            <input
+              placeholder="email@example.com"
+              name="email"
+              type="email"
+              class="field"
+              id="mail"
+              required>
```

Comment:
```suggestion
            <input
              placeholder="email@example.com"
              name="email"
              type="email"
              class="field"
              id="mail"
              required
            >
```
Выравнивай закрывающую скобку с открывающей